region/lowernorfair/east.json

```diff
@@ -556,6 +907,50 @@
                       ]
                     }
                   ]
+                },
+                {
+                  "name": "Ice Assist",
+                  "notable": false,
+                  "requires": [
+                    "h_canNavigateHeatRooms",
+                    "canUseFrozenEnemies",
+                    "h_CrouchJumpDownGrab",
+                    {"heatFrames": 700}
+                  ],
+                  "obstacles": [
+                    {
+                      "id": "A",
+                      "requires": [
+                        "h_canUsePowerBombs",
+                        {"heatFrames": 50}
+                      ]
+                    }
+                  ]
+                },
+                {
+                  "name": "IBJ",
+                  "notable": false,
+                  "requires": [
+                    "h_canNavigateHeatRooms",
+                    {"or": [
+                      "h_canBombAboveIBJ",
+                      "h_canUsePowerBombs"
```

Comment:
or `double bomb jump` or `jump into ibj`?
This will double count the PB. one for the shot block, one for the PB block A

Reply:
I think using the PB at the right height is too difficult and in that case you would use `canBombAboveIBJ`
`jump into ibj` is part of hjbless.  It could be moved here.